android/source/src/main/kotlin/com/gu/source/components/pager/PagerProgressBar.kt

```diff
@@ -150,7 +168,10 @@ fun PagerProgressBar(
                                 (pagerState.currentPage + 1).coerceAtMost(pagerState.pageCount - 1)
                             }
                         }
-                        pagerState.animateScrollToPage(page)
+                        pagerState.animateScrollToPage(
+                            page = page,
+                            animationSpec = pageSlideAnimationSpec,
```

Comment:
Key change 1: Use the provided animation spec when changing pages from progress buttons. Set to `spring(stiffness = Spring.StiffnessMediumLow)`